### DISCUSSION TITLE
Dev mode - Custom FBW

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#3690] [General]
Hi,

what is the difference between developer mode and the custom Mode? Cause the intersesting thing is the TCA thrust is working correct in the custome mode but not in the developer mode.

Greets Klaus

**Comment by MStallen** [comment]
I've the same question. Would be **very** nice if the throttle would also work in de dev mode. I mean really nice

**Reply by tracernz** [reply]
Throttle works fine with both versions. MS updated the default throttle profile for TCA a couple of updates ago to "fix" it so maybe try going back to the default profile if you're having trouble on the dev version. It's still a little touchy at the MCT detent though.

**Reply by CptBayer** [reply]
Hmmm ... not for me.Only in custom mode. But what is the difference between this two variations?

Greets Klaus

**Comment by pickels98** [comment]
Same problem. Trottle works only correct in custom mode. In Dev mode the works folder is missing the cfg. file
